tools/boards.txt.py

```diff
@@ -295,6 +295,31 @@
                   'Note that since ESP8285 has SPI flash memory internally connected in DOUT mode, pins 9 and 10 may be used as GPIO / I2C / PWM pins.',
                   ],
     }),
+    ( 'agruminolemon', {
+        'name': 'Lifely Agrumino Lemon v4 (WT8266-S1 Module)',
```

Comment:
Is the "WT8266-S1 module" portion really necessary?  This is the longest board name by a big margin...

Reply:
@earlephilhower Hi, is good only "Lifely Agrumino Lemon v4" ???

Reply:
I have modified, can you check if everything is ok?